plugins/woocommerce/client/legacy/js/admin/wc-enhanced-select.js

```diff
@@ -304,6 +304,98 @@ jQuery( function( $ ) {
 
 					$( this ).selectWoo( select2_args ).addClass( 'enhanced' );
 				});
+
+				// Ajax category search boxes
+				$( ':input.wc-taxonomy-term-search' ).filter( ':not(.enhanced)' ).each( function() {
+					var return_format = $( this ).data( 'return_id' ) ? 'id' : 'slug';
+					var minimumInputLength = 3;
+
+					if ( $( this ).data( 'minimum_input_length' ) !== undefined ) {
+						minimumInputLength = $( this ).data( 'minimum_input_length' );
+					}
```

Comment:
Can we use here a ternary operator as the line before?
It would look like this:
```
var minimumInputLength = $( this ).data( 'minimum_input_length' ) !== undefined ? $( this ).data( 'minimum_input_length' ) : 3;
```
Another good option would be:
```
var minimumInputLength = $( this ).data( 'minimum_input_length' ) ?? 3;
```

Reply:
Fixed as part of: https://github.com/woocommerce/woocommerce/pull/34744/commits/9cf74f8da34fd29d5ac415bf6901edc952bb3547